test/integration/pages/home.py

```diff
@@ -26,7 +26,7 @@ class Home(Base):
     _new_entry_locator = (By.CSS_SELECTOR,
                           'section menu '
                           'button.{}:nth-child(1)'.format(
-                            munged_class_name('button')))
+                              munged_class_name('button')))
```

Comment:
Any reason for this change?

Reply:
When looking at this and the changes below, the new indentations seem more consistent, so I'm happy with it.

Reply:
I didn't realize I included these changes. I was only trying to push the new documentation file.

I got under-indentation errors when running flake8, which is the reason for these edits.